axiom/util_system.c

```diff
@@ -39,21 +45,115 @@ char* nr_system_get_hostname(void) {
   return nr_strdup(hn);
 }
 
-nr_system_t* nr_system_get_system_information(void) {
+/* Populates the nr_system_t struct with libc info if available. */
+static void nr_system_get_system_libc(nr_system_t* sys) {
+  const char* libc_version = NULL;
+
+  if (NULL == sys) {
+    return;
+  }
+
+#if defined(__GLIBC__)
+  libc_version = gnu_get_libc_version();
+#else
+/* Currently unable to extract MUSL version. */
```

Comment:
I would omit the `#else` case here entirely. MUSL devs have made it aggressively clear they don't ever intend to add identifiers into their implementation like `__GLIBC__`

Reply:
removed here [a327cf5](https://github.com/newrelic/newrelic-php-agent/pull/1140/commits/a327cf5950fdadedce1d9afc4269c31944d68246)